### PR TITLE
Add dropdown menu

### PR DESCRIPTION
**Pull request by KevinCawley**
Change html and scss to allow for a dropdown menu.

**Description**

When the proper format is followed, a dropdown menu will be created with links to the appropriate items.
**Motivation and context**

Feature. 
**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [x] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [x] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [x] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Comment by KevinCawley** [comment]
This is a link to the docs:
https://kevincawley.github.io/tardis-sn.github.io/branch/dropdown_menu/

**Comment by KevinCawley** [comment]
https://kevincawley.github.io/tardis-sn.github.io/branch/dropdown_menu/

File: assets/sass/imports/_header.scss [hunk 1]
```diff
@@ -251,3 +251,85 @@
...
+        top: 5px;
+        left: 30px;
+        padding: 0px 0px 12px 0px;
+    }
+
+    .dropdown .dropdown-item {
+        display: flex !important;
+        position: relative;
+        padding: 0px 1px !important;
+        box-shadow: 0px 0px 0px rgba(0,0,0,0)!important;
```
Comment (jaladh-singhal) [inline assets/sass/imports/_header.scss:330]: Why are you using `!important` - it's highly discouraged to use it unless you've very good reason to?

File: assets/sass/imports/_header.scss [hunk 2]
```diff
@@ -56,7 +56,7 @@
 
   .site-navigation {
     .current-menu-item {
-      color: $gray-500;
+      color: $gray-500 !important;
```
Comment (jaladh-singhal) [inline assets/sass/imports/_header.scss:59]: Same here?

File: assets/sass/imports/_header.scss [hunk 3]
```diff
@@ -251,3 +251,85 @@
...
+    display: none;
+    min-width: max-content;
+    height: 36px;
+    background-color: #fff;
+    margin: 0 auto;
+}
+
+.dropdown-item:hover {
+    background-color: #f1f1f1;
+}
```
Comment (jaladh-singhal) [inline assets/sass/imports/_header.scss:300]: Also prepend them with `.menu-header` to make sure the styles are applied only for dropdown in header

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]
This looks great! But I've few requests to make code better.

Also, the community section in footer shows icons incorrectly now with your changes - probably you are targetting css classes on html wrongly?

File: assets/sass/imports/_header.scss (hunk 2 above)
Comment (KevinCawley) [reply assets/sass/imports/_header.scss:59]: Removed

File: assets/sass/imports/_header.scss [hunk 4]
```diff
@@ -251,3 +251,85 @@
...
+
+.menu-header .dropdown .dropdown-item {
+    display: none;
+    position: relative;
+    padding: 5px 10px;
+    right: 10px;
+    vertical-align: middle;
+    align-items: center;
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.3);
```
Comment (jaladh-singhal) [inline assets/sass/imports/_header.scss:286]:
```suggestion
    box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.4);
```
It looks better with more blur

File: assets/sass/imports/_header.scss [hunk 5]
```diff
@@ -251,3 +251,85 @@
...
+.menu-header .dropdown-item {
+    display: none;
+    min-width: max-content;
+    height: 36px;
+    background-color: #fff;
+    margin: 0 auto;
+}
+
+.menu-header .dropdown-item:hover {
+    background-color: #f1f1f1;
```
Comment (jaladh-singhal) [inline assets/sass/imports/_header.scss:298]:
```suggestion
    background-color: #f6f6f6;
```
Lighten gray a bit

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]
There few minor things

File: assets/sass/imports/_header.scss (hunk 5 above)
Comment (KevinCawley) [reply assets/sass/imports/_header.scss:298]: I think it's harder to see this way. I have trouble.

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**